packages/component-library/src/components/plmg-button/plmg-button.scss

```diff
@@ -1,4 +1,13 @@
 @use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+@use 'sass:math';
+
+// Functions
+// ------------------------------
+@function centerIconFontSize($parentFontSize, $parentLineHeight) {
+  @return math.div($parentFontSize * $parentLineHeight, 100%);
```

Comment:
I'm not sure this is scaling appropriately.  

In a center icon extra large button the svg is coming out as 32.4x32.4px while we expect 42x42px.
As such the whole button is too small. It is 66.4x66.4px rather than 80x80px.

Reply:
Good catch, thanks! :detective:

Reply:
That's fixed.
My logic was correct but the multiplier for extra-large was not. And the padding for small was not.